CovertInfiltration/Src/CovertInfiltration/Classes/X2InfiltrationModTemplate.uc

```diff
@@ -10,20 +10,17 @@
 class X2InfiltrationModTemplate extends X2DataTemplate;
 
 // Number of hours this item adds to the Covert Action clock
-var int InfilModifier;
+var int HoursAdded;
 // When equipping this item, items with an ItemCat matching MultCategory

```

Comment:
Add a blank line before full line comments (this and `// How much [...]` later). Again, for readability